src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -141,7 +141,13 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 		if (this.editable && !this._isInAccessibilityMode) {
 			this._editableDropdown.values = this.getValues();
 			if (this.value) {
-				this._editableDropdown.value = this.getSelectedValue();
+				if (this._editableDropdown.value !== this.getSelectedValue()) {
```

Comment:
the event should be fired by the editableDropdown itself, right? here is just a angular wrapper of it.